### PR TITLE
compare room

### PR DESCRIPTION
**Pull request by minhld1709**

### CODE REVIEW

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -23,4 +23,160 @@
...
+  while(compare_house_ids.length != 0){
+    m = compare_house_ids.pop();
+    if(compare_house_ids.length != 0)
+      fullUrl += m + ",";
+    if(compare_house_ids.length == 0)
+      fullUrl += m;
+  }
+  window.location = fullUrl;
+}
+
```
Comment (samnangcattor) [inline app/assets/javascripts/application.js]: create  new file javascript for your script  and  remove line from line 26 to 91 and line 93 to 182 , No remove line 92 , and please use require

File: app/views/compare/addresses/_address.html.erb [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    <% if address.washing_machine %><p>- <%= t "facility.washing_machine" %></p><% end %>
+    <% if address.television %><p>- <%= t "facility.television" %></p><% end %>
+    <% if address.network %><p>- <%= t "facility.network" %></p><% end %>
+    <% if address.table %><p>- <%= t "facility.table" %></p><% end %>
+    <% if address.chair %><p>- <%= t "facility.chair" %></p><% end %>
+  </td>
+  <td><%= address.address %></td>
+  <td>
+    <% if address.parking? %>
+      <p>has</p>
```
Comment (samnangcattor) [inline app/views/compare/addresses/_address.html.erb]: In80 and line 28 too

File: app/assets/javascripts/compare.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
+$(document).ready(function(){
+  $(".check_box_compare").prop("checked", false);
+  $(".fancybox").fancybox({
+    openEffect  : 'none',
```
Comment (bapboy18) [inline app/assets/javascripts/compare.js]: try "

File: app/assets/javascripts/compare.js [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+function transmission(){
+  var m;
+  var fullUrl = "/compare/addresses";
+  fullUrl += "?address_ids=";
+  while(compare_house_ids.length != 0){
+    m = compare_house_ids.pop();
+    if(compare_house_ids.length != 0)
+      fullUrl += m + ",";
+    if(compare_house_ids.length == 0)
```
Comment (bapboy18) [inline app/assets/javascripts/compare.js]: why "why(compare_house_ids.length != 0)"  and 
still have "compare_house_ids.length == 0"

and you should have !== and === instead != and ==

File: app/views/compare/addresses/_address.html.erb [hunk 5]
```diff
@@ -0,0 +1,36 @@
+<tr>
+  <td class="list_img">
+    <% address.images.each do |img| %>
+      <%= link_to image_tag(img.photo.url), img.photo.url,
+      title: "#{address.title}",
```
Comment (samnangcattor) [inline app/views/compare/addresses/_address.html.erb]: You should use  address.title for replace  "#{address.title}"

Comment (bapboy18) [inline app/views/compare/addresses/_address.html.erb]: indent

File: app/views/compare/addresses/index.html.erb [hunk 6]
```diff
@@ -0,0 +1,18 @@
+<div id="compare_wrapper">
+  <table class="table table-bordered" id="compare">
+    <thead>
+      <tr>
+        <th>Images</th>
```
Comment (bapboy18) [inline app/views/compare/addresses/index.html.erb]: I18n

File: app/assets/javascripts/compare.js (hunk 4 above)
Comment (minhld1709) [inline app/assets/javascripts/compare.js]: why?

File: app/views/compare/addresses/_address.html.erb (hunk 5 above)
Comment (minhld1709) [inline app/views/compare/addresses/_address.html.erb]: dont understant? samnang??

File: app/assets/javascripts/compare.js [hunk 7]
```diff
@@ -0,0 +1,59 @@
+$(document).ready(function(){
```
Comment (bapboy18) [inline app/assets/javascripts/compare.js]: need more space

File: app/views/compare/addresses/index.html.erb [hunk 8]
```diff
@@ -0,0 +1,18 @@
+<div id="compare_wrapper">
+  <table class="table table-bordered" id="compare">
+    <thead>
+      <tr>
+        <th><%= t "Images" %></th>
+        <th><%= t "Square" %></th>
+        <th><%= t "Price" %></th>
+        <th id="facility">Facility</th>
```
Comment (bapboy18) [inline app/views/compare/addresses/index.html.erb]: I18n

**Pull request closed without merging**